Contributing.md

```diff
@@ -30,10 +30,19 @@ This project uses clang-format (stable branch) to check for common style issues.
 
 In most cases, clang-format can and **should** be used to automatically reformat code and solve most formatting issues.
 
-- To run clang-format on all staged files:
+It can be installed using your distro package manager (on Linux) or from [LLVM binary packages](http://releases.llvm.org/download.html) for the latest version.
+
+- To run clang-format on all staged files, use the following command:
   ```
-  git diff --cached --name-only | egrep '[.](cpp|h|mm)$' | xargs clang-format -i
+  git diff --cached --name-only | egrep '[.](cpp|h|mm)$' | xargs clang-format -style=file -i
   ```
+  
+  - If you can't execute clang-format on Windows, try locating `clang-format.exe` in the LLVM installation and place it into the `...Git\usr\bin` directory.
+  
+- VS2017 users can install the [clang-format extension](https://marketplace.visualstudio.com/items?itemName=HansWennborg.ClangFormat#overview).
+
+   - Preferrably run the VSIX installer while Visual Studio is not running.
+   - Supports "Format When Save" feature, but it's disabled by default.
```

Comment:
Still don't think we have to mention this, as this is purely an editor preference.

Reply:
I heard that to be the way people use this plugin, so I thought if someone doesn't know about such a feature, wouldn't go trying the plugin out to find it. It's a plugin feature, not standard of VS.

Is this such a big nuisance that you really feel strongly about it?